packages/mocks/atlantic-forces.mock.ts

```diff
@@ -309,6 +310,7 @@ export const forces: ForceData[] = [
     "overview": "Blue force.",
     "roles": [
       {
+        "roleId": "r12345CO",
```

Comment:
@lebaphi - I _think_ it would be healthy for the three `roleId` values in this data were unique.